### DISCUSSION TITLE
Disappearance of the app from Google Play

### DISCUSSION THREAD

**Discussion by david-lev** [LibChecker/LibChecker#727] [Q&A]
Hi!
First, thanks for the app, I use it all the time. 
I noticed that the app disappeared (maybe even removed) from Google Play. What happened?

**Comment by zhaobozhen** [comment]
> Hi!
> First, thanks for the app, I use it all the time. 
> I noticed that the app disappeared (maybe even removed) from Google Play. What happened? 

![image](https://user-images.githubusercontent.com/25247117/235350286-64b2c000-4336-4224-9681-f6d766c158d9.png)

The Play Store updated its privacy policy, but I didn't receive advance notice of the change, resulting in it being taken down from the store. Don't worry, it's not a big deal 😌

**Comment by david-lev** [comment]
@zhaobozhen Thanks for the response, hope it will be resolved soon :)
